plugin/package.json

```diff
@@ -40,6 +40,7 @@
     "prepare": "npm run build"
   },
   "dependencies": {
+    "@gatsby-cloud-pkg/merlin-synchronizer": "^0.3.6",
```

Comment:
Adding it as a dep of the plugin _should_ make it resolvable to builds, but we can do tests with auto-install using feature flags later